clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -3345,12 +3345,9 @@ public void testDeleteConsumerGroupOffsetsRetryBackoff() throws Exception {
     }
 
     @Test
-    public void testDeleteConsumerGroupOffsets() throws Exception {
-        // Happy path
-
+    public void testDeleteConsumerGroupOffsetsResponseIncludeCoordinatorErrorAndNoneError() throws Exception {
```

Comment:
Add a test to include coordinator error and other None errors in all partition response. We should retry it, too.